tests/e2e/utils/RequstHandler.ts

```diff
@@ -0,0 +1,62 @@
+import { TestConstants, RequestType } from '..';
+import axios from 'axios';
+import querystring from 'querystring';
+
+export class RequestHandler {
+    async processRequest(reqType: RequestType, url: string) {
+        let response;
+        // maybe this check can be moved somewhere else at the begining so it will be executed just once
+        if (TestConstants.TS_SELENIUM_MULTIUSER === true) {
+            let authorization = 'Authorization';
+            axios.defaults.headers.common[authorization] = 'Bearer ' + await this.getCheBearerToken();
+        }
+        switch (reqType) {
+            case RequestType.GET: {
+                response = await axios.get(url);
+                break;
+            }
+            case RequestType.DELETE: {
+                response = await axios.delete(url);
+                break;
+            }
+            default: {
+                throw new Error('Unknown RequestType: ' + reqType);
+            }
+        }
+        return response;
+    }
+
+    async getCheBearerToken(): Promise<string> {
+        let params = {};
+
+        let keycloakUrl = TestConstants.TS_SELENIUM_BASE_URL;
```

Comment:
I think we should suppose, that the system under test is Che. 
Would be nice though if it would be somehow extracted (into variable?) and could be overriden in downstream.